README.md

```diff
@@ -32,6 +32,14 @@ Rendering Examples:
 Source code for this example can be found [here](examples/cairo_example.py).
 
 
+Install from source:
+```
+$ git clone https://github.com/curtacircuitos/pcb-tools.git
+$ cd pcb-tools
+$ make

```

Comment:
Make is not strictly necessary to install. Think we can remove that as it may cause trouble with non Unix users.